filesystems/xfs/xfstests/README.md

```diff
@@ -0,0 +1,28 @@
+xfstests test suite
+=====================================
+Wrapper for xfstests test suite, see http://git.kernel.org/?p=fs/xfs/xfstests-dev.git;a=summary \
+Test Maintainer: [Xiong Murphy Zhou](mailto:xzhou@redhat.com)
+
+How to run it
+-------------
+Make sure you have at least 25G free disk space in this partition
+```bash
+$ df -H .
+```
+
+Clone the tests-beaker repository
+```bash
+$ git clone https://github.com/CKI-project/tests-beaker.git
```

Comment:
Since the README.md is already in the directory with the test, we don't need any instructions on how to clone the repo.

Reply:
Good point, I removed it